src/views/EditorView/Editor/Editor.tsx

```diff
@@ -61,11 +62,16 @@ class Editor extends React.Component<IProps, IState> {
         this.canvas.addEventListener("mousedown", this.mouseDownEventBus);
 
         const {imageData, size ,activeLabelType} = this.props;
-        this.loadImage(imageData);
+
+        ImageLoader.add(async () => {
+            await this.loadImage(imageData);
+        });
+
         this.resizeCanvas(size);
         this.primaryRenderingEngine = new PrimaryEditorRenderEngine(this.canvas, this.imageRectOnCanvas);
         this.mountSupportRenderingEngine(activeLabelType);
-        this.fullCanvasRender()
+        this.fullCanvasRender();
+        setTimeout(() => ImageLoader.run(), 10);
```

Comment:
no właśnie... dlatego napisałem że to bieda kolejkowanie bo tak zasugerowałem żeby zrobić :/ 
na tę chwilę to rozwiązanie może być. ale wypadałoby wymyślić jakieś lepsze